code/__HELPERS/mobs.dm

```diff
@@ -155,18 +155,18 @@ GLOBAL_LIST_INIT(skin_tones, sortList(list(
 	)))
 
 GLOBAL_LIST_INIT(skin_tone_names, list(
-	"african1" = "African 1",
-	"african2" = "African 2",
+	"african1" = "Caramel",
+	"african2" = "Espresso",
 	"albino" = "Albino",
-	"arab" = "Arab",
-	"asian1" = "Asian 1",
-	"asian2" = "Asian 2",
-	"caucasian1" = "White 1",
-	"caucasian2" = "White 2",
-	"caucasian3" = "White 3",
-	"indian" = "Indian",
-	"latino" = "Latino",
-	"mediterranean" = "Mediterranean",
+	"arab" = "Mocha",
+	"asian1" = "Ivory",
+	"asian2" = "Beige",
+	"caucasian1" = "Porcelain",
+	"caucasian2" = "Sand",
+	"caucasian3" = "Peach",
+	"indian" = "Amber",
+	"latino" = "Sienna",
+	"mediterranean" = "Olive",
```

Comment:
This is officially too woke for me

Reply:
They're the real names of those skin tones. I'm also pretty sure this isn't the first time I've seen someone push for a change like this. I can't remember why it wasn't merged.

Reply:
It's definitely not better, for me beige is no different than sand, and previous color names are much easier to sort through.

If I want to be caucasian-skinned, only three causasian tan levels to choose from, each in increasing strength level.

Reply:
It doesn't require color literacy:

![2021-09-10T19-47-09](https://user-images.githubusercontent.com/35135081/132962303-b90b646f-67eb-4138-a32c-ca98ae754969.png)

Reply:
> I can't remember why it wasn't merged.

Because the last one didn't migrate savefiles--this one is only display names, and so requires no savefile changing.

I cleared it through oranges who was the one who closed the PR last time, and he said I could if I also showed the color next to it.

Reply:
@Mothblocks well, how do I know which ones of them are supposed to be caucasian? It's all mixed up.

Reply:
Also, this is olive: 
![image](https://user-images.githubusercontent.com/1516236/132962855-b257b230-632b-4d80-b2d3-fc5fd95f5fbd.png)

Reply:
Anyway, not blocking the PR with this, but just saying that it would probably be better to keep the old names and sort them by increasing darkness, but grouped by ethnicity (which will avoid mixing up differences in hue).

Reply:
Forget one /s and everyone loses their minds

Reply:
![image](https://user-images.githubusercontent.com/35135081/132965813-e113dbe2-60ac-4ceb-a61d-5ee8f908b9c6.png)

I changed the names anyway to be more intuitive 😄